packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -45,6 +45,7 @@ contract DeployConfig is Script {
     uint256 public l2GenesisBlockBaseFeePerGas;
     uint256 public gasPriceOracleOverhead;
     uint256 public gasPriceOracleScalar;
+    bool public enableGovernance;
     uint256 public eip1559Denominator;
```

Comment:
I believe there is a "fundDevAccounts" config option, it would be nice to parse that and then if true, have a function that `vm.deal` a bunch of ether to a set of accounts. The accounts could be the default hh/anvil addresses. You can see them by running `anvil`

Reply:
[Updated!](https://github.com/ethereum-optimism/optimism/pull/9292/files#diff-2b05d0be8d7d8de7870ac51225a0728adeb9a6df9819eaa654c455a23b8f509eR381-R388)